### PR TITLE
Add Python3 compatibility

### PR DESCRIPTION
**Pull request by StephenKrewson**
Ran 2to3 against the repo. Added some additional explanations in the README.

I am actively using this package in my HT research! Needed to have it compatible with Python3.6.

Thanks!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 from distutils.core import setup
```
Comment (rlmv) [inline setup.py]: Could you remove these changes to setup.py?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,35 @@ A simple interface for the HathiTrust APIs. The package contains basic classes a
 [data api]: http://www.hathitrust.org/data_api
 [solr api]: http://wiki.htrc.illinois.edu/display/COM/Solr+Proxy+API+User+Guide
 
+## UPDATES in Fork
```
Comment (rlmv) [inline README.md]: These comments are very useful, but shouldn't be in the README - they belong in the commit message.

File: hathitrust_api/bib_api.py [hunk 3]
```diff
@@ -1,7 +1,11 @@
+from __future__ import absolute_import
+from __future__ import division
```
Comment (rlmv) [inline hathitrust_api/bib_api.py]: I don't think the `division` and `print_function`  futures are needed in this file.

File: hathitrust_api/constants.py [hunk 4]
```diff
@@ -1,4 +1,6 @@
-
+from __future__ import absolute_import
```
Comment (rlmv) [inline hathitrust_api/constants.py]: I don't think any `__futures__` imports are needed here.

File: hathitrust_api/data_api.py [hunk 5]
```diff
@@ -1,8 +1,11 @@
+from __future__ import absolute_import
+from __future__ import division
```
Comment (rlmv) [inline hathitrust_api/data_api.py]: `print_function` and `division` are not needed here.

File: hathitrust_api/solr_api.py [hunk 6]
```diff
@@ -1,12 +1,15 @@
+from __future__ import absolute_import
+from __future__ import division
```
Comment (rlmv) [inline hathitrust_api/solr_api.py]: `division` is not needed here

**Comment by rlmv** [comment]
I can also add the `requests` and `requests-oauthlib` dependencies to `setup.py` so they are automatically installed.

File: setup.py (hunk 1 above)
Comment (StephenKrewson) [reply setup.py]: Will do! Making requested changes now.

**Comment by StephenKrewson** [comment]
Hi @rlmv --just made those changes! Went ahead and added install_requires to setup.py. Can get rid of that if it's not correct. Thanks for the great feedback and package!

**Comment by rlmv** [comment]
Thanks @StephenKrewson, this is now merged.

**Comment by StephenKrewson** [comment]
Great! Thank you!

**Comment by rlmv** [comment]
I just added the package to PyPi so you can now install it using `pip install hathitrust-api`. Cheers!

**Pull request merged**